gnocchi/opts.py

```diff
@@ -64,6 +64,15 @@ def __getitem__(self, key):
     cfg.PortOpt('port',
                 default=8041,
                 help="Port to listen on"),
+    cfg.StrOpt('uwsgi-mode',
+               default='http',
+               choices=["http", "http-socket", "socket"],
+               help="""Socket type to use for uwsgi:
+* http: support HTTP/1.1 and keepalive,
+  but not chunked encoding (eg: influxdb)
+* http-socket/socket: support chunked encoding, but require a upstream HTTP
```

Comment:
That does not tell what's the difference between http-socket and socket thought

Reply:
They are identical for us, while it depends on how you configure apache/nginx:
* http-socket: nginx<->uwsgi talks http protocol
* socket: nginx<->uwsgi talks uwsgi protocol

That's why I didn't put more detail.